content/security/helmet.md

```diff
@@ -1,6 +1,6 @@
 ### Helmet
 
-[Helmet](https://github.com/helmetjs/helmet) can help protect your app from some well-known web vulnerabilities by setting HTTP headers appropriately. Generally, Helmet is just a collection of 14 smaller middleware functions that set security-related HTTP headers (read [more](https://github.com/helmetjs/helmet#how-it-works)).
+[Helmet](https://github.com/helmetjs/helmet) can help protect your app from some well-known web vulnerabilities by setting HTTP headers appropriately. Generally, Helmet is just a collection of 15 smaller middleware functions that set security-related HTTP headers (read [more](https://github.com/helmetjs/helmet#how-it-works)).
```

Comment:
```suggestion
[Helmet](https://github.com/helmetjs/helmet) can help protect your app from some well-known web vulnerabilities by setting HTTP headers appropriately. Generally, Helmet is just a collection of few smaller middleware functions that set security-related HTTP headers (read [more](https://github.com/helmetjs/helmet#how-it-works)).
```

I don't think this number is relevant to the NestJS docs (as users can discover that on Helmet docs instead). Also, if they change that number in the future we won't need to change it here as well. Do you agree?

Reply:
Let's just make it `a collection of smaller middleware functions`. I think @micalevisk has a good point that if the number changes we would have to update again, or risk having conflicting numbers